src/main/java/domain/card/Hands.java

```diff
@@ -0,0 +1,71 @@
+package domain.card;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static domain.Game.FIRST_DRAW_NUMBER;
+
+public class Hands {
```

Comment:
블랙잭에서 한 명의 플레이어가 가지는 패를 `Hand`라고 부릅니다. 일급 컬렉션이라서 복수형 네이밍을 선택하신 것 같은데, `Hand`가 이미 복수형의 의미를 내포하고 있으니 아래와 같이 바꾸는게 좋지 않을까요?

```suggestion
public class Hand {
```

---
> 40 CARDS [countable]
> a) the playing cards given to one person in a game

참고: https://www.ldoceonline.com/ko/dictionary/hand